.github/actions/integration-tests/run-ci-stage1

```diff
@@ -436,14 +449,13 @@ for userenv in ${CI_ACTIVE_USERENVS}; do
 
                 # Only run all-in-one json test if the runfile exists for the endpoint
                 # e.g.: ci-oslat-run-file-k8s-alma8.json
-                RUN_FILE="${SCRIPT_DIR}/run-file/ci-oslat-run-file-${CI_ENDPOINT}-${userenv}.json"
+                RUN_FILE="${SCRIPT_DIR}/run-file/ci-${scenario}-run-file-${CI_ENDPOINT}-${userenv}.json"
                 if [ -f "${RUN_FILE}" ]; then
-                    sed -i "s|CI_REPEAT_RUNS|${CI_REPEAT_RUNS}|g" ${RUN_FILE}
-                    sed -i "s|USERENV|${userenv}|g" ${RUN_FILE}
-                    sed -i "s|SCRIPT_DIR|${SCRIPT_DIR}|g" ${RUN_FILE}
-                    sed -i "s|CONTROLLER_IP|${CONTROLLER_IP}|g" ${RUN_FILE}
-                    sed -i "s|CI_ENDPOINT_HOST|${CI_ENDPOINT_HOST}|g" ${RUN_FILE}
-                    sed -i "s|CI_ENDPOINT_USER|${CI_ENDPOINT_USER}|g" ${RUN_FILE}
+                    source ${SCRIPT_DIR}/prepare-run-file 
+                    replace_value userenv ${userenv} '."endpoints"[0]."userenv" = $userenv'
```

Comment:
Same comment as above for userenv -- it's already in the filename apparently so why would you need to set/change it?

Reply:
Well, it is not used today because we wanted to limit the number of tests for run-files. So the strategy was to create a ci-${scenario}-run-file-${userenv}.json for only few cases and expand it later.  We can eventually remove the userenv check that decides if the test will run or not and replace the content of userenv. In this case we can edit the json and set userenv: default for all scenarios, and let the scenario for a particular userenv to edit the json live. For now, I'd like to keep the jq so we can easily switch later.